app/javascript/components/artists/ArtistProfile.jsx

```diff
@@ -171,22 +256,42 @@ class ArtistProfile extends React.Component {
             {works.map(work => {
               if (this.getAvailability(activeFilter).includes(work.availability)) {
               return (
-                    <WorkColumnPanel work={work} key={work.id} hideArtistName={true}>
-                  {canEditProfile &&
-                    <div className="work-action-wrapper mb2">
-                      <Button type="hover-button" onClick={() => this.updateWork(work.id)}>
-                        <FontAwesomeIcon className="white" icon={faEdit} />
-                        <h4 className="ml2 white">Edit</h4>
+                <div>
+                {(!work.hidden || this.props.userType == "admin" || (this.props.user != null && (this.props.user.account_id == this.props.artist.account_id))) &&
+                  <WorkColumnPanel work={work} key={work.id} hideArtistName={true}>
+                    {canEditProfile &&
+                      <div className="work-action-wrapper mb2">
+                        <Button type="hover-button" onClick={() => this.updateWork(work.id)}>
+                          <FontAwesomeIcon className="white" icon={faEdit} />
+                          <h4 className="ml2 white">Edit</h4>
+                        </Button>
+                        {(work.availability == 'active') &&
+                          <Button className="ml2" type="hover-button" onClick={() => this.deleteWork(work.id)}>
+                            <FontAwesomeIcon className="white" icon={faTrash} />
+                            <h4 className="ml2 white">Delete</h4>
+                          </Button>
+                        }
+                      </div>
+                    }
+                    {this.props.userType == "admin" &&
+                      <div>
+                      {work.hidden == false &&
+                      <Button className="ml2" type="hover-button" onClick={() => this.hideWork(work.id)}>
```

Comment:
Left-most button again 
```suggestion
                      <Button type="hover-button" onClick={() => this.hideWork(work.id)}>
```